src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -49,16 +52,16 @@ class CromwellApiServiceActor(val workflowManagerActorRef : ActorRef, swaggerSer
 trait CromwellApiService extends HttpService with PerRequestCreator  {
   val workflowManagerActorRef : ActorRef
 
-  val workflowRoutes = queryRoute
+  val workflowRoutes = queryRoute ~ submitRoute
 
   @ApiOperation(value = "Query for workflow based on workflow id.",
     nickname = "query",
     httpMethod = "GET",
-    produces = "application/json",
-    response = classOf[WorkflowStatusResponse]
+    produces = "application/json"
+    //, response = classOf[WorkflowStatusResponse]
```

Comment:
This (and the next one) are the lines that were causing all the strange compiler problems.  Not sure how to fix.  The impact right now is the swagger doesn't describe the output message format.